cpp/src/aztec/common/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Collect our common/*.hpp files and include in installation
+file(GLOB_RECURSE HEADER_FILES *.hpp)
+target_sources(
+    barretenberg_headers
+    INTERFACE
+    FILE_SET HEADERS
+    FILES ${HEADER_FILES}
+)
```

Comment:
Is this only necessary here because common is not a `barretenberg_module`?

Reply:
Correct!